### ISSUE TITLE
Let's Encrypt SSL certificates are being linked to an expired root certificate

### ISSUE THREAD

**Issue by MLeFrench** [opnsense/core#5251]
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Describe the bug**

New generated/re-issued SSL certificates with Let's Encrypt via the ACME Client are linked to an expired R3 Root Certificate.

*Tip*: to validate your setup was working with the previous version, use opnsense-revert (https://docs.opnsense.org/manual/opnsense_tools.html#opnsense-revert)

**To Reproduce**

1. Go to Services 
2. Click on ACME Client
3. Go to Certificates
4. Issue new Certificate that's with an ACME Account linked to Let's Encrypt
5. Validate the Certificate when created with a free service eg: https://www.sslshopper.com

**Expected behavior**

A valid SSL Certificate is generated with no issues in its chain.

**Describe alternatives you considered**

Using another SSL certificate provider

**Screenshots**

![Screenshot (188)](https://user-images.githubusercontent.com/13226834/135449622-5a038669-1584-40fe-971a-8a7f4cbff3da.png)

**Relevant log files**

n/a

**Additional context**

The R3 root certificate expired yesterday:

https://scotthelme.co.uk/lets-encrypt-old-root-expiration/
https://techcrunch.com/2021/09/21/lets-encrypt-root-expiry/

**Environment**

Software version used and hardware type if relevant, e.g.:

OPNsense 21.7.3_1-amd64
FreeBSD 12.1-RELEASE-p20-HBSD
OpenSSL 1.1.1l 24 Aug 2021

**Comment by fichtner** [comment]
Wrong repo and duplicate.

**Comment by MLeFrench** [comment]
Thanks @fichtner, 🤦‍♂️ found it in plugins https://github.com/opnsense/plugins/issues/2550

**Comment by ssbarnea** [comment]
In fact I am quite worried as opnsense updates failed to due to invalid certificate.

**Comment by AdSchellevis** [comment]
@ssbarnea likely these issues are due to a certificate locally installed, just try to update from a fresh install (works like a charm from here). Most of this is caused due to how let's encrypt handles their expired certificate (I assume you can't access their website either from the same install as it distributes the same chain)

**Comment by ssbarnea** [comment]
My OpenSense box was configured to auto-update and I am running 21.7.3_1 which right now is unable to get any updates, from any mirror! I doubt all of them are using the new certificate.
```
***GOT REQUEST TO CHECK FOR UPDATES***
Currently running OPNsense 21.7.3_1 (amd64/OpenSSL) at Fri Oct  1 09:45:52 BST 2021
Fetching changelog information, please wait... Certificate verification failed for /O=Digital Signature Trust Co./CN=DST Root CA X3
4435996811264:error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed:/usr/src/crypto/openssl/ssl/statem/statem_clnt.c:1915:
fetch: https://pkg.opnsense.org/FreeBSD:12:amd64/21.7/sets/changelog.txz.sig: Authentication error
Updating OPNsense repository catalogue...
...
```

The funny bit is that my router has an ACME certificate configured on it for it own interface and I was able to use ACME client to regenerate certificates. That part worked but the router itself is not able to get any updates and I don't really understand why this happens.

Wasn't the "default" mirror option designed to provide resiliency for the case when one of few mirrors go down? I tried picking 4-5 and I still get the same kind of errors,... and interestingly they all involve `https://pkg.opnsense.org`. I guess that its certificate needs the new intermediate which for some reason was not loaded by the router (no idea why considering auto-update was enabled).

I guess I need to manually add the new intermediate ACME certificate to fix it but I am unable to find it.
